src/test/java/it.uniroma3.mat.extendedset/intset/ImmutableConciseSetTest.java

```diff
@@ -1823,6 +1823,116 @@ public void testComplement9()
     verifyComplement(expected, testSet, length);
   }
 
+
+  /**
+   * Complement of a null set with a length
+   */
+  @Test
+  public void testComplement9_31()
```

Comment:
Maybe we can combine all your tests into one with a set of specific lengths we'd like to test, and then also add a bunch of randomly generated lengths between 0 and Integer.MAX_VALUE?